src/pudl_archiver/archivers/ferc/xbrl.py

```diff
@@ -286,14 +330,12 @@ async def archive_year(
     metadata = {}
     archive_path = output_dir / f"ferc{form_number}-xbrl-{year}.zip"
 
+    # Track taxonomies referenced by all filings for archival
+    taxonomies_referenced = set()
```

Comment:
non-blocking musing: instead of looking to see which taxonomies are *actually* referenced in all the filings, does it make sense to just hard-code "these are all the FERC taxonomies for each form, archive all of them"? Less in-code logic, but we have to update the map every time a new taxonomy becomes relevant.

Reply:
I figured since we already have to look inside the filings to put the referenced taxonomy in metadata, that we might as well use that to get the list of taxonomies. I'm not particularly tied to that though.